lms/static/sass/xblocks/image_explorer.scss

```diff
@@ -177,10 +177,12 @@ body.new-theme {
               &:hover {
                 background-color: transparent !important;
               }
-              i{
+              &:after {
                 font-family: "Material Icons";
+                content: "\e14c";
```

Comment:
I think we were removing content attributes as part of Accessibility.

Reply:
@nasirhjafri you are Right, but it's not a new style that we are including, it just reverts back the code. So it, not a primary requirement of this ticket, where we are using .sr class for the Screen Reader.

Reply:
Yes, please use the HTML element instead.

Reply:
I'll let someone else review, @msaqib52  can you please review it? Thanks

Reply:
@hussnain2 if there are design differences between stock and mckinsey IE feedback close button than it is ok to add it as after element, As long as the button is focusable by keyboard and screen readers read this button.

Reply:
@msaqib52  yes you got the point, Actually, there is a Design difference between stock and Mckinsey IE feedback close button. meanwhile, it's still focusable.

Reply:
ok thanks in that case it is ok, Approving the PR